src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -237,74 +278,24 @@ const AppWrapper = (props: { children: ReactNode }) => {
 
     try {
       await Agent.agent.start();
+      setIsOnline(true);
     } catch (e) {
-      // @TODO - foconnor: Should specifically catch the error instead of all, but OK for now.
-      setAgentInitErr(true);
-      // eslint-disable-next-line no-console
-      console.error(e);
-      return;
+      const errorStack = (e as Error).stack as string;
+      // If the error is failed to fetch with signify, we retry until the connection is secured
+      if (/SignifyClient/gi.test(errorStack)) {
+        Agent.agent.bootAndConnect().then(() => {
+          setIsOnline(Agent.agent.getKeriaOnlineStatus());
+        });
+      } else {
+        throw e;
+      }
     }
     dispatch(setPauseQueueIncomingRequest(true));
```

Comment:
We still need to handle this properly now. I think we can remove this line from here.

I think we need to:
- Call `setPauseQueueIncomingRequest(!isOnline)` in the `isOnline` useEffect IF `authentication.loggedIn` from Redux is `true`.
- In `stateCache.ts` - in `login: (state)` only update `state.queueIncomingRequest` IF we are online.
- I think `logout: (state)` is OK to leave as it is.

cc @jimcase what do you think? Hope there aren't any extreme timing cases here